components/dashboard/Header.js

```diff
@@ -0,0 +1,31 @@
+import Link from 'next/link';
+import Button from '../buttons/Button';
+
+export default function Header() {
+  return (
+    <div className="flex justify-between">
+      <div className="lg:flex lg:justify-between">
+        <div className="max-w-xl">
+          <h2 className="text-4xl leading-10 font-extrabold text-gray-900 sm:text-5xl sm:leading-none sm:tracking-tight lg:text-6xl">
+            <span className="shapeup-animated-gradient">Dashboard</span>
+          </h2>
+          <p className="mt-5 text-xl leading-7 text-gray-500">
+            Lorem ipsum dolor sit amet.{' '}
```

Comment:
😄

Reply:
you got me. 😆

Reply:
This is still unresolved. Did you resolve by mistake? 🤔

Reply:
@fmvilas  Sorry, it is updated now.